### PR TITLE
fix matrix size when zooming out screen and downloading to png

### PR DESCRIPTION
**Pull request by JackVCurtis**

### CODE REVIEW

File: views/components/Matrix.d3.js [hunk 1]
```diff
@@ -53,15 +53,15 @@ export default class Matrix {
   }
 
   validDay = (d) => d.day >= this.startDay && d.day <= this.lastDayForFilter
-
+
```
Comment (mikestone14) [inline views/components/Matrix.d3.js]: 👾

File: views/pages/GraphPage.jsx [hunk 2]
```diff
@@ -156,15 +157,16 @@ const GraphPage = () => {
...
       let svgString = new XMLSerializer().serializeToString(updatedSvgElement)
-      let svgUrl =
-        'data:image/svg+xml; charset=utf8, ' + encodeURIComponent(svgString)
 
       const kanvas = canvasRef.current
       kanvas.width = updatedSvgElement.getBBox().width
       kanvas.height = updatedSvgElement.getBBox().height
 
+      let svgUrl =
+        'data:image/svg+xml; charset=utf8, ' + encodeURIComponent(svgString.replace('<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%">', `<svg xmlns="http://www.w3.org/2000/svg" width="${kanvas.width}" height="${kanvas.height}">`))
```
Comment (mikestone14) [inline views/pages/GraphPage.jsx:166]: so using percentage widths doesn't work to fill up the height/width? That's surprising to me. Will this implementation continue to work as the user changes the screen size?

Comment (JackVCurtis) [reply views/pages/GraphPage.jsx:166]: Percentage widths work fine in the browser but when we render on a `canvas` it stops working 🤷 . And yes these does work when the screen size changes.

File: views/components/Matrix.d3.js (hunk 1 above)
Comment (JackVCurtis) [reply views/components/Matrix.d3.js]: 0c63744

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**